build.gradle

```diff
@@ -61,3 +69,12 @@ configure(sourceProjects()) {
     }
 
 }
+
+def addCloudExtensionDependency(proj) {
+
+    if (proj.rootProject.ext.cloudType == "google") {
+        proj.dependencies { compile project(':extensions:cloud:portability-cloud-google') }
```

Comment:
do we need this case statement? Can we require that a cloud type be set and just use "compile project(':extensions:cloud:portability-cloud-$cloudType')"

Reply:
Good point. I'll add that in